packages/@coorpacademy-components/src/molecule/banner/index.js

```diff
@@ -49,7 +62,8 @@ Banner.propTypes = {
   firstCTA: PropTypes.func,
   firstCTALabel: PropTypes.string,
   secondCTALabel: PropTypes.string,
-  secondCTA: PropTypes.func
+  secondCTA: PropTypes.func,
+  temporary: PropTypes.bool
```

Comment:
You don't need to add this attribute.. you can juste use the style 
type: PropTypes.oneOf(['information', 'error', 'successTemporary', 'errorTemporary]) , cuz now we don't have the type success only

Reply:
@jomaora a ajouter le type 'success' donc je l'ai garder et j'ai ajouter une nouvelle prop 'temporary'

Reply:
personnellement ça ne me choque pas qu'on ait cette property temporary

Reply:
@jomaora @DjamelSoualmi on garde la prop donc ?